db/migrate/20171022032108_all_the_predictions.rb

```diff
@@ -0,0 +1,14 @@
+class AllThePredictions < ActiveRecord::Migration
+  def change
+    # add_column :crops, :perennial, :boolean, default: false
+    # add_column :crops, :median_time_to_first_harvest, :interval
+    # add_column :crops, :median_time_to_last_harvest, :interval
+    # add_column :crops, :median_time_to_finished, :interval
```

Comment:
i want to save these on crops, rather than calculate on every planting/harvest edit. then we can update all plantings (probably in a rake task), instead of just the one we are editing

Reply:
Do we want mean or median here?